tests/integration/conftest.py

```diff
@@ -64,4 +77,56 @@ def time_series(request, cognite_client):
     create_subscription_in_cdf(timeseries_set, sub_name, cognite_client)
     yield timeseries_set
     remove_time_series_data(timeseries_set, cognite_client)
-    cognite_client.time_series.subscriptions.delete(sub_name)
\ No newline at end of file
+    cognite_client.time_series.subscriptions.delete(sub_name)
+
+@pytest.fixture(scope="session")
+def test_space(cognite_client: CogniteClient):
+    space = cognite_client.data_modeling.spaces.retrieve(SPACE_ID)
+    if space is None:
+        new_space = SpaceApply(SPACE_ID, name="Integration Test Space", description="The space used for integration tests.")
+        space = cognite_client.data_modeling.spaces.apply(new_space)
+    yield space
+    cognite_client.data_modeling.spaces.delete(spaces=[SPACE_ID])
+    
+
+@pytest.fixture(scope="session")
+def test_dml():
+    return (RESOURCES / "movie_model.graphql").read_text()
+
+@pytest.fixture(scope="function")
+def test_model(cognite_client: CogniteClient, test_space: Space, test_dml: str):
+    movie_id = DataModelId(space=test_space.space, external_id="Movie", version="1")
+    created = cognite_client.data_modeling.graphql.apply_dml(
```

Comment:
Get error if Space with the same name already exists. Need to consider how to handle this, maybe just an Assert that it doesn't exist?

Reply:
Added some code to clear out any existing data models in the space, please test

Reply:
Still fails, old space still exists.

Reply:
I'll just add an assert statement for now